apps/www/content/docs/installation.mdx

```diff
@@ -71,6 +73,7 @@ Here's how I structure my app. I use Next.js, but this is not required.
 - The rest of the components such as `<PageHeader />` and `<MainNav />` are placed in the `components` folder.
 - The `lib` folder contains all the utility functions. I have a `utils.ts` where I define the `cn` helper.
 - The `styles` folder contains the global CSS.
+- The `components.json` contains the components configuration.
```

Comment:
- The `components.json` contains the components configuration.
- The `components.json` contains configuration for the CLI.